### ISSUE TITLE
Cancellation

### ISSUE THREAD

**Issue by czaplej** [zhaosiyang/axios-observable#4]
I think Cancellation isn't working well. because it's not cancell te request to the server or I'm not right?

**Comment by jsalamander** [comment]
Can confirm that calling `unsubscribe()` does not cancel the request.

```javascript
 this.queryExecuting = true;
      const requestObservable = BackendService.runSql(this.sqlQuery).subscribe(
        response => {
          this.result = response.data;
          this.headers = [];
          for (let k in this.result[0]) {
            this.headers.push({ text: k, value: k });
          }
          this.sqlQuerySuccess = this.sqlQuery;
        },
        error => {
          this.error = error.response.data.error || error.response.data.message;
          this.queryExecuting = false;
        },
        () => {
          this.queryExecuting = false;
        }
      );
      requestObservable.unsubscribe();
```

the sent request isn't cancelled afterwards...

**Comment by zhaosiyang** [comment]
@janfriedli Sorry for the late reply. Currently it's not cancelling. And we still need to use axios cancel token to do cancelling. 
Thanks for pointing that out, I will make cancelling happen when unsubscribing.

**Comment by zhaosiyang** [comment]
Issue fixed by #6, thus closing the issue. It's now in new version v1.1.0. Thanks for reporting the problem
